src/main/java/uk/gov/hmcts/reform/ccd/test/stubs/service/token/JWTokenGenerator.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.reform.ccd.test.stubs.service.token;
+
+import io.jsonwebtoken.JwtBuilder;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import java.security.Key;
+import java.util.Date;
+import java.util.Map;
+import java.util.UUID;
+
+public final class JWTokenGenerator {
+
+    private JWTokenGenerator() {
+    }
+
+    public static String generateToken(String privateKeyFileName,
+                                       String issuer,
+                                       long ttlMillis,
+                                       Map<String, Object> claims) {
+        final SignatureAlgorithm signatureAlgorithm = SignatureAlgorithm.RS256;
+
+        final long nowMillis = System.currentTimeMillis();
+        final Date now = new Date(nowMillis);
+
+        final Key signingKey = KeyUtil.getPrivateKey(privateKeyFileName);
+
+        final JwtBuilder builder = Jwts.builder()
+            .setId(UUID.randomUUID().toString())
```

Comment:
i guess it should map with wiremock jwks response. So, better use some fixed value.

Reply:
private key should be kept secret right.. we don't need to expose that outside.. if we provide the json file then we can access that from outside...
wiremock response should include only public keys... our existing /o/jwks endpoint exposing public key only....